src/api/openapi.rs

```diff
@@ -58,6 +59,8 @@ impl Api<Object<PodSpec, PodStatus>> {
     }
 }
 
+impl Log for Api<Object<PodSpec, PodStatus>> {}
```

Comment:
Rather than having a one line impl for one resource object that every user would need to import the trait of, could we instead:

- have `log` always public but `unimplemented!()` or `{}` by default
- have an extra `impl Api<Object<PodSpec, PodStatus>>` that overrides `log`

Then you have one less import required for users.

Reply:
I implements log operation in this way, because I hope `trait Log` can be reused by other developers when they implement their crds

Reply:
Yeah, that makes sense. Either way is fine, it would just be easier for users of the Api to not have to import the extra trait.